### ISSUE TITLE
MeshCtrl has no output

### ISSUE THREAD

**Issue by prononext** [Ylianst/MeshCentral#5198]
I am trying to use MeshCtrl on my Meshcentral Dev Instance and **I get no output, just a new line**. 

I was trying to do it on a production instance and had the same error so I started up a clean dev instance to see if there are any changes. The dev instance is has just one user "admin" with password "admin".

**My Command Variations all result in a empty new line after about 20 seconds**
1. ```node node_modules/meshcentral/meshctrl.js serverinfo --url wss://localhost:443 –-loginuser admin –-loginpass admin```
2. ```node node_modules/meshcentral/meshctrl.js serverinfo --url wss://localhost:443 –-loginuser "admin" –-loginpass "admin"```

**All also tested with**
1. ```wss://localhost```
2. ```wss://localhost:80```
3.  ```wss://serverip:80```
4.  ```wss://serverip:443```
5. ```wss://serverip```
6. No --url parameter
7.  And single quotes on User and Password
8. meshctrl.js in .node_modules/meshcentral/meshctrl.js and also on other windows and linux client in network with same result
9. With several other meshctrl commands which are shown when I run ```node meshctrl.js```

**Server Software (please complete the following information):**
 - OS: Ubuntu Server 22
 - Meshcentral Version: 1.0.6
 - Node: 18.16.0 (tested on 12 + 14 also)

**config.json file (standard and unchanged for transparent testing on DEV instance**
```
{
  "$schema": "https://raw.githubusercontent.com/Ylianst/MeshCentral/master/meshcentral-config-schema.json",
  "__comment1__": "This is a simple configuration file, all values and sections that start with underscore (_) are ignored. Edit a section and remove the _ in front of the name. Refer to the user's guide for details.",
  "__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
  "settings": {
    "_cert": "myserver.mydomain.com",
    "_WANonly": true,
    "_LANonly": true,
    "_sessionKey": "MyReallySecretPassword1",
    "_port": 443,
    "_aliasPort": 443,
    "_redirPort": 80,
    "_redirAliasPort": 80
  },
  "domains": {
    "": {
      "_title": "MyServer",
      "_title2": "Servername",
      "_minify": true,
      "_newAccounts": true,
      "_userNameIsEmail": true
    }
  },
  "_letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "myemail@mydomain.com",
    "names": "myserver.mydomain.com",
    "skipChallengeVerification": true,
    "production": false
  }
}
```
**```info``` output on web terminal**
```
    "meshVersion": "v1.1.6",
    "nodeVersion": "v18.16.0",
    "runMode": "LAN mode",
    "productionMode": true,
    "database": "NeDB",
    "platform": "linux",
    "arch": "x64",
    "pid": 778,
    "uptime": 2992.658648767,
    "cpuUsage": {
        "user": 7450248,
        "system": 430951
    },
    "memoryUsage": {
        "rss": 149254144,
        "heapTotal": 37785600,
        "heapUsed": 35823256,
        "external": 31110089,
        "arrayBuffers": 30204671
    },
    "warnings": [],
    "allDevGroupManagers": []
```
**```versions``` output on web terminal**
```
    "node": "18.16.0",
    "acorn": "8.8.2",
    "ada": "1.0.4",
    "ares": "1.19.0",
    "brotli": "1.0.9",
    "cldr": "42.0",
    "icu": "72.1",
    "llhttp": "6.0.10",
    "modules": "108",
    "napi": "8",
    "nghttp2": "1.52.0",
    "nghttp3": "0.7.0",
    "ngtcp2": "0.8.1",
    "openssl": "3.0.8+quic",
    "simdutf": "3.2.2",
    "tz": "2022g",
    "undici": "5.21.0",
    "unicode": "15.0",
    "uv": "1.44.2",
    "uvwasi": "0.0.15",
    "v8": "10.2.154.26-node.26",
    "zlib": "1.2.13"
```

**Output of my webtrace from when testing the command from different hosts and variations**
```
webrequest | (127.0.0.1) /control.ashx/.websocket
httpheaders | GET, /control.ashx/.websocket, [object Object]
webrequest | (127.0.0.1) /control.ashx/.websocket
httpheaders | GET, /control.ashx/.websocket, [object Object]
webrequest | (10.10.0.6) /images/images16.png
httpheaders | GET, /images/images16.png, [object Object]
webrequest | (10.10.0.6) /images/trash.png
httpheaders | GET, /images/trash.png, [object Object]
webrequest | (192.168.1.123) /images/trash.png
httpheaders | GET, /images/trash.png, [object Object]
dispatch | DispatchEvent, *,server-users,user//admin
webrequest | (10.10.0.6) /images/meshicon50.png
httpheaders | GET, /images/meshicon50.png, [object Object]
webrequest | (10.10.0.6) /images/icon-relay.png
httpheaders | GET, /images/icon-relay.png, [object Object]
webrequest | (10.10.0.6) /images/icons16.png
httpheaders | GET, /images/icons16.png, [object Object]
webrequest | (10.10.0.6) /control.ashx/.websocket
httpheaders | GET, /control.ashx/.websocket, [object Object]
dispatch | DispatchEvent, *
webrequest | (127.0.0.1) /control.ashx/.websocket
httpheaders | GET, /control.ashx/.websocket, [object Object]
webrequest | (127.0.0.1) /control.ashx/.websocket
httpheaders | GET, /control.ashx/.websocket, [object Object]
dispatch | DispatchEvent, *

```

**Comment by OutbackMatt** [comment]
Try it without the meshcentral.js

Like 
node node_modules/meshcentral serverinfo --url wss://localhost:443 –-loginuser "admin" –-loginpass "admin"

**Comment by prononext** [comment]
That command is just starting meshcentral, which is already started on my server. 

**output**
```
MeshCentral HTTP redirection server running on port 81.
MeshCentral v1.1.6, LAN mode.
MeshCentral HTTPS server running on port 444.
```

**I want to run meshctrl.js to interact witht the Meshcentral server through my  terminal on the server itself and from other remote machines**

Is there anyone who had the same problem, because meshcentral in general is running like a charm, just meshctrll will not work somehow. I watched all [Ylianst](https://github.com/Ylianst) great videos on youtube about that theme, but nothing mentioned about any extra setting.

**Comment by prononext** [comment]
After a long trial and error I noticed that I need to run:

```
sudo service meshcentral stop
node node_modules/meshcentral/meshcentral.js --user admin
```

Then connect one time with meshctrl.js from another machine while the previous command is executing
Then quit the previous command and do

```
sudo service meshcentral start
```

***After that the meshctrl seems to work. ***

I did that all while installing and before I created the system service, but after restoring any backup I needed to run it again, to get meshctrl up and running. 

I cannot explain that behaviour... anyhow good that it is working now....
Hope it will help someone.
